scripts/date-snapshot/date-snapshot.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
+
+# Script to download one or several git repos at the last commit before a specified date
+# Takes inputs specified via command line or in a file
+# Created by June Knauth (VMware) <june.knauth@gmail.com>, 2022-06-22
+
+package=date-snapshot
+
+download_checkout(){
+  IN="${1}"
+  # $fields=(${IN//;/ }) 
+  IFS=';' read -ra ADDR <<< "$IN"
+  
```

Comment:
Add check if the destination directory is empty, or may be just delete it if already exists to ensure `git clone` will not be confused.

Reply:
I'd argue not stomping on the directory is more better?  Might need an option of "--preclean" or something to do deletes if it finds something it doesn't like or expect